src/AppBundle/Resources/views/base.html.twig

```diff
@@ -17,6 +17,7 @@
     <link href="{{ absolute_url(asset('css/lib/sticky-footer-navbar.css')) }}" type="text/css" rel="stylesheet" />
     <link href="{{ absolute_url(asset('css/aaplus-styles.css')) }}" type="text/css" rel="stylesheet" />
   {% endblock %}
+  <script id="CookieConsent" src="https://policy.app.cookieinformation.com/uc.js"     data-culture="DA" type="text/javascript"></script>
```

Comment:
Remove excess spaces in tag between src and data-culture
src="https://policy.app.cookieinformation.com/uc.js"     data-culture="DA"